src/renderer/inc/RenderEngineBase.hpp

```diff
@@ -23,16 +23,6 @@ namespace Microsoft::Console::Render
 {
     class RenderEngineBase : public IRenderEngine
     {
-    public:
-        ~RenderEngineBase() = 0;
-
-    protected:
-        RenderEngineBase();
-        RenderEngineBase(const RenderEngineBase&) = default;
-        RenderEngineBase(RenderEngineBase&&) = default;
-        RenderEngineBase& operator=(const RenderEngineBase&) = default;
-        RenderEngineBase& operator=(RenderEngineBase&&) = default;
-

```

Comment:
Curious: why don't we need these? Shouldn't we generally try to define these copy constructors? Or is it that because they're set to default, we're not actually doing anything here.

Reply:
By changing `bool _titleChanged;` to `bool _titleChanged = false;` we can remove the default constructor. The other 2 constructors and the 2 assignment operators only existed (and subsequently got removed), because [C26432](https://docs.microsoft.com/en-us/cpp/code-quality/c26432?view=msvc-170) nags if you define a default constructor without defining all other special ones. I really don't like C26432...